ztag/schema.py

```diff
@@ -106,6 +106,31 @@ class CensysString(WhitespaceAnalyzedString):
     "timestamp":DateTime(),
 })
 
+ztag_dsa_params = SubRecord({
+    "p":IndexedBinary(),
+    "q":IndexedBinary(),
+    "g":IndexedBinary(),
+    "y":IndexedBinary(),
+})
+
+ztag_ecdsa_public_key = SubRecord({
```

Comment:
It's currently used only by SSH. I did not make `parsed_certificate` reuse the `ecdsa_public_key` definition because [this comment](https://github.com/zmap/ztag/blob/ff309ab3b68f07df0cad76e204ea9c4e2ac4044b/ztag/schema.py#L189) indicates a special case affects `parsed_certificate`'s ECDSA public key schema.